src/intervals/arithmetic.jl

```diff
@@ -155,6 +155,25 @@ end
 
 //(a::Interval, b::Interval) = a / b    # to deal with rationals
 
+if VERSION >= v"0.6.0-dev"
+    const filter = Iterators.filter
+end
+
+if VERSION < v"0.5"
+    min(x) = x
+    max(x) = x
```

Comment:
No, I mean do you need to be extending a base method on base types? If these are backporting something that worked in later versions of Julia, is it available in Compat?